plugins/modules/vlan_member.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+#
+# GNU General Public License v3.0+ (see https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vlan_member
+short_description: Configure a SONiC VLAN port memberships
+version_added: "0.1.2"
+description: Manage VLAN port memberships in SONiC
+options:
+    vlanid:
+        description: The VLAN to manage the interface membership for
+        required: true
+        type: int
+    state:
+        description: Whether the interface should be attached to the VLAN or not
+        default: present
+        type: str
+        choices: [present, absent]
+    interface:
+        description: The interface to attach to the VLAN
+        required: true
+        type: str
+    tagging_mode:
```

Comment:
I lazily copied how it looks in the configdb :-)
How about `untagged` with false/off being the default, it would make it clearer when reading playbooks which interfaces are untagged or not (assuming most users uses tagged interfaces)?

Reply:
My problem with `untagged` is that it's a negative which tend to be harder to read (c.f. `untagged: no` vs. `tagged: yes`).

As for what the default should be, personally I would probably make it a required field to avoid confusion. I know from experience how often I forget the `-u` flag in SONiC when I want to make a port untagged - I wish they made the CLI require specifying both so you don't have to remember which is the default.

I'll leave both these choice to you since it's your PR.

Reply:
I went with your suggestion as I agree that making the option explicit is better